.github/workflows/opsim-test.yml

```diff
@@ -1,10 +1,10 @@
 name: opsim-test
 on:
-  # Triggers the workflow on push or pull request events but only for the main branch
+  # Triggers the workflow on push or pull request events
   push:
-    branches: [ main ]
+    branches: [ maxim-ci ]
```

Comment:
I would just make this run on all the branches. Surely, it only matters on the main branch, but you probably want to know that your changes have broken the unit tests, before comitting to main, no?

Reply:
Yes, I agree. Will add.